src/main/java/emu/grasscutter/server/event/game/SceneBlockLoadedEvent.java

```diff
@@ -0,0 +1,14 @@
+package emu.grasscutter.server.event.game;
+
+import emu.grasscutter.server.event.types.ServerEvent;
+import emu.grasscutter.scripts.data.SceneBlock;
+
+public final class SceneBlockLoadedEvent extends ServerEvent {
+    public SceneBlock block;
```

Comment:
consider making final or using `@Getter` and `@Setter`

Reply:
thanks; changed members to private and used @Getter/@Setter